packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js

```diff
@@ -20,14 +20,15 @@ function InjectDropdownMenuItem({
 	onKeyDown,
 	...rest
 }) {
+	const Renderers = Inject.getAll(getComponent, { MenuItem });
```

Comment:
Why use Renderers for react bootstrap component ?

Reply:
Because they are registred now